hooks/cache_location.py

```diff
@@ -98,12 +98,14 @@ def _get_cache_root(self, project_id, pipeline_configuration_id):
 
         # get site only; https://www.foo.com:8080 -> www.foo.com
         base_url = urlparse.urlparse(tk.shotgun.base_url)[1].split(":")[0]
-        
+
         # now structure things by site, project id, and pipeline config id
-        cache_root = os.path.join(root, 
-                                  base_url, 
-                                  "project_%d" % project_id,
-                                  "config_%d" % pipeline_configuration_id)
+        cache_root = os.path.join(
```

Comment:
It will break the cached thumbnails and queries for the desktop, yes. There's no way to get around that. We used to use the template project id. Now we'll be using something more telling of that particular project. What do you think?

However, it is possible that if they have overriden this hook in a shared core between desktop and their project and used %d, then they will get a crash. The only way to get around that would be to have get_project_id return 0, but others felt that 0 might be problematic elsewhere.